Makefile

```diff
@@ -102,13 +109,8 @@ push: build
 		PLUGIN_NAME=`echo "$${PLUGIN_FULL_NAME}" | cut -d '/' -f 3-`; \
 		PLUGIN_VERSION=`yq '.plugin_version' $${plugin_dir}/buf.plugin.yaml`; \
 		echo "Pushing plugin: $${plugin}"; \
-		if [[ "$(DOCKER_ORG)" != "bufbuild" ]]; then \
+		if [[ "$(DOCKER_ORG)" = "ghcr.io/bufbuild" ]]; then \
 			$(DOCKER) pull $(DOCKER_ORG)/plugins-$${PLUGIN_OWNER}-$${PLUGIN_NAME}:$${PLUGIN_VERSION} || exit 1; \
```

Comment:
We only want to pull an image if `DOCKER_ORG` points to the remote registry. Otherwise, this will fail on local builds.